README.md

```diff
@@ -6,3 +6,112 @@ AiiDA plugin that simplifies working with pseudo potentials.
 [![Docs status](https://readthedocs.org/projects/aiida-pseudo/badge)](http://aiida-pseudo.readthedocs.io/)
+
+
+## Getting started
+
+The easiest way of getting started using `aiida-pseudo` is to use the command line interface that ships with it.
+For example, to install a configuration of the [SSSP](https://www.materialscloud.org/discover/sssp/table/efficiency), just run:
+
+    aiida-pseudo install sssp
+
+The version, functional or protocol can be controlled with various options; use `aiida-pseudo install sssp --help` to see their description.
+Installed pseudopotential families can be listed using:
+
+    aiida-pseudo list
+
+Any pseudopotential family installed can be loaded like any other `Group` using the `load_group` utility.
+Once loaded, it is easy to get the pseudopotentials for a given element or set of elements:
+
+    from aiida.orm import load_group
+    family = load_group('SSSP/1.1/PBE/efficiency')
+    pseudo = family.get_pseudo(element='Ga')  # Returns a single pseudo
+    pseudos = family.get_pseudos(elements=('Ga', 'As'))  # Returns a dictionary of pseudos where the keys are the elements
+
+If you have a `StructureData` node, the `get_pseudos` method also accepts that as an argument to automatically retrieve all the pseudopotentials for required for that structure:
+
+    structure = load_node()  # Load the structure from database or create one
+    pseudos = family.get_pseudos(structure=structure)
+
+If you use `aiida-quantumespresso` the `pseudos` dictionary returned by `get_pseudos` can be directly used as an input for `PwCalculation`s.
+
+## Design
+
+The plugin is centered around two concepts: pseudopotentials and pseudopotential families.
+The two concepts are further explained below, especially focusing on how they are implemented in `aiida-pseudo`, what assumptions are made and what the limitations are.
+
+### Pseudopotentials
+
+Pseudopotentials are implemented as [data plugins](https://aiida-core.readthedocs.io/en/latest/topics/data_types.html#creating-a-data-plugin) and the base class is `PseudoPotentialData`.
+Pseudopotentials are assumed to be defined by a single file on disk and represent a single chemical element.
+As such, each pseudopotential node, _has_ to have two attributes: the md5 of the file that it represents and the symbol of chemical element that the pseudopotential describes.
+The latter follows IUPAC naming conventions as used in the module `aiida.common.constants.elements` of `aiida-core`.
+
+The `PseudoPotentialData` functions as a base class and does not represent an actual existing pseudopotential format.
+It _is_ possible to create a family of `PseudoPotentialData` nodes, but since the element cannot be parsed from the file content itself, it will have to be done from the _filename_, so the filename _has_ to have the format `CHEMICAL_SYMBOL.EXTENSION`, for example `Ar.pseudo`.
+Since the file does not represent an actual pseudopotential, this class is mostly useful for development.
```

Comment:
Since the `PseudoPotentialData` class does not have any features tailored to specific formats, as do the classes which inherit from it (e.g. `UpfData` for UPF-formatted pseudopotentials), this class is mostly useful for development.